docs/assets/scss/schema-docs.scss

```diff
@@ -330,4 +330,34 @@
       @include u-text('bold');
     }
   }
+}
+
+
+.no-anchor-xslt {
+  // : :after
+  font-style: normal;
+  font-variant: normal;
+  font-kerning: auto;
+  font-optical-sizing: auto;
+  font-feature-settings: normal;
+  font-variation-settings: normal;
+  font-weight: normal;
+  font-stretch: normal;
+  font-size: 1em;
+  line-height: inherit;
+  font-family: anchorjs-icons;
+  position: absolute;
+  margin-left: -1em;
+  padding-right: 0.5em;
+}
+
+.no-anchor-xslt:hover {
+  text-decoration: underline;
+  transform: scale(1.06);
+  border-image: "⚓";
+}
+
+.no-anchor-xslt :hover::before {
+  font-family: anchorjs-icons;
+  background-image: "⚓";
 }
```

Comment:
Blocking: expanding the above recommendation.

```suggestion

```

Reply:
I knew that font-family: anchorjs-icons; would include ⚓. Whatever family we use by default and includes ⚓ is fine with me.

Reply:
> I knew that font-family: anchorjs-icons; would include ⚓. Whatever family we use by default and includes ⚓ is fine with me.

I think we have discussed this before: why do we need the anchor? It does not render with the accompanying Metaschema PR and we will re-generate all documentation for other released OSCAL reference docs, so we do not need this anymore (even as an interim gap measure). Am I remembering correctly?